src/components/editor/Bindings/SchemaEditCommands/DiscoveredSchema.tsx

```diff
@@ -0,0 +1,60 @@
+import { Terminal } from '@mui/icons-material';
+import { Box, Typography } from '@mui/material';
+import SingleLineCode from 'components/content/SingleLineCode';
+import { useEditorStore_persistedDraftId } from 'components/editor/Store/hooks';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+function DiscoveredSchemaCommands() {
+    const intl = useIntl();
+
+    const persistedDraftId = useEditorStore_persistedDraftId();
+
+    const pullDraftCommand = intl.formatMessage(
+        {
+            id: 'workflows.collectionSelector.schemaEdit.discoveredCollection.command1',
+        },
+        {
+            draftId: persistedDraftId,
+        }
+    );
+
+    const pushLocalEditsCommand = intl.formatMessage({
+        id: 'workflows.collectionSelector.schemaEdit.discoveredCollection.command2',
+    });
+
+    return (
+        <>
+            <Box
```

Comment:
The header for this _CLI Commands_ popper should be shared between this component and `ExistingSchemaCommands` (i.e., _src/components/editor/Bindings/SchemaEditCommands/ExistingSchema.tsx_).